flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/RemoteInputChannel.java

```diff
@@ -439,6 +439,14 @@ public int getInitialCredit() {
 		return initialCredit;
 	}
 
+	public int unsafeGetExclusiveBufferUsed() {
```

Comment:
`unsynchronized` instead of `unsafe` to match existing `unsynchronizedGetNumberOfQueuedBuffers`?

Reply:
ExclusiveBuffers